src/backend/api.ts

```diff
@@ -16,23 +16,24 @@ interface Params {
 
 export function setupApi(app: express.Application) {
 	app.use('/api/createSession', (req: express.Request, res: express.Response) => {
-		createSession((req.query as Params).captcha, req.connection.remoteAddress)
+		return createSession((req.query as Params).captcha, req.connection.remoteAddress)
```

Comment:
FYI if you wanted to you could use ES6 arrow functions that don't require the "return" operator, but you must also remove the curly braces for them to work.

Reply:
Undesrood